source/includes/opts/telemetry-statement.rst

```diff
@@ -0,0 +1,3 @@
+By default, ``mongosync`` collects anonymous, aggregated usage data to
```

Comment:
I think we should explicitly mention that telemetry data would be sent to [Segment](https://segment.com/)

Reply:
gotcha that makes sense. I'm cool with leaving it out then